frontend/language/src/en.json

```diff
@@ -2,8 +2,12 @@
   "dashboard": "Dashboard",
   "dashboard.delete.form": "Delete form",
   "dashboard.download": "Download data",
+  "dashboard.empty.message": "Press the button on the right to create your first form!",
```

Comment:
The button `New form` is good described,😊 so I think no need to tell the user where the button is because it is placed in a good place. What do you think if we say "Dashboard is empty,  add a form" or maybe just "Dashboard is empty".

Reply:
I see what you mean 😊 
It is now changed to ""Dashboard is currently empty."